roles/ssl/tasks/main.yml

```diff
@@ -86,7 +92,8 @@
       -in {{keystore_path}} \
       -passin pass:{{keystore_storepass}} \
       -nodes -nocerts \
-      -out {{key_path}}
+      -out {{key_path}} && \
+    chown -R {{uid}}:{{gid}} {{key_path}}
   args:
     # Assuming the key is correct if exists
     creates: "{{key_path}}"
```

Comment:
WDYT about removing all the chown commands and putting this as the final task:
```
- name: Create Directory for SSL Files
  file:
    path: "{{component_output_dir}}"
    recursive: true
    owner: "{{ uid }}"
    group: "{{ gid }}"
```

Reply:
Works for me! component_output_dir will be valid for both local and remote deployments right?